docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -218,7 +221,20 @@ tcld login
 tcld apikey delete --id <api-key-id>
 ```
 
-## Manage API Keys for Service Accounts {#serviceaccount-api-keys}
+### Rotate an API key
+
+Temporal API keys automatically expire.
+Their maximum life is 90 days.
+Follow these steps to rotate API keys:
+
+1. Create a new key.
+   You may reuse key names if that helps.
+1. Allow both keys to be active at the same time.
+1. Switch to the new key and start using it.
```

Comment:
nit: Switch clients to load the new key and start using it.

Reply:
Replacing x 2.